pkg/util/oom/oom_linux.go

```diff
@@ -60,20 +73,22 @@ func applyOOMScoreAdj(pid int, oomScoreAdj int) error {
 		pidStr = strconv.Itoa(pid)
 	}
 
-	oomScoreAdjPath := path.Join("/proc", pidStr, "oom_score_adj")
 	maxTries := 2
+	oomScoreAdjPath := path.Join("/proc", pidStr, "oom_score_adj")
+	value := strconv.Itoa(oomScoreAdj)
 	var err error
 	for i := 0; i < maxTries; i++ {
-		_, readErr := ioutil.ReadFile(oomScoreAdjPath)
-		if readErr != nil {
-			err = fmt.Errorf("failed to read oom_score_adj: %v", readErr)
-		} else if writeErr := ioutil.WriteFile(oomScoreAdjPath, []byte(strconv.Itoa(oomScoreAdj)), 0700); writeErr != nil {
-			err = fmt.Errorf("failed to set oom_score_adj to %d: %v", oomScoreAdj, writeErr)
-		} else {
-			return nil
+		if err = ioutil.WriteFile(oomScoreAdjPath, []byte(value), 0700); err != nil {
+			exists, perr := processExists(pid)
```

Comment:
does the caller of this function (applyoomscoreadjcontainer) require similar treatement or is it absolved? Noticed it translates cgroups to pids by peeking in sysfs (I think). Or is that not the right callstack even.